lib/link_expansion/hashed_edition.rb

```diff
@@ -0,0 +1,37 @@
+module LinkExpansion::HashedEdition
+  def edition_hash(values)
+    return nil unless values.present?
+    hash = values.is_a?(Array) ? hash_for(values) : values
```

Comment:
Yeah this can be an array of values from the AR `pluck` https://github.com/alphagov/publishing-api/pull/976/files#diff-d21d7da8d7d146227416a146fde15973R51 but can also be `Edition#attributes` from here https://github.com/alphagov/publishing-api/pull/976/files#diff-d21d7da8d7d146227416a146fde15973R23